plugins/org.jboss.tools.vpe/src/org/jboss/tools/vpe/editor/VpeEditorPart.java

```diff
@@ -1171,7 +1171,7 @@ public void updatePartAccordingToPreferences() {
 	
 	public void updateSelectionBar(boolean isSelectionBarVisible) {
 		if (selectionBar != null) {
-			selectionBar.setVisible(isSelectionBarVisible);
+			selectionBar.setRealBarVisible(isSelectionBarVisible);
```

Comment:
Ok, I've changed setVisible to setRealBarVisible also in code which is commented out. Thanks!